src/averageAges.js

```diff
@@ -7,20 +7,24 @@
  * function calculates average age only for men who died in this century
  *
  * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
+ * Divide year of kid's death by 100: Math.ceil(kid.died / 100)
  *
  * @param {object[]} people
  * @param {number} century - optional
  *
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+  const isCenturyFilter = people.filter(kid => (century
+    ? kid.sex === 'm' && Math.ceil(kid.died / 100) === century
+    : kid.sex === 'm'));
```

Comment:
you check 'century' on every iteration. You can check the 'century' only 1 time and choose what filter to apply (the same in next functions)